mobile/src/Student/screens/Game/PhaseOneBasicGamePlay/index.js

```diff
@@ -197,17 +202,19 @@ const PhaseOneBasicGamePlay = ({
                     GameSessionState.CHOOSE_CORRECT_ANSWER ? (
                     <>
                         <Text style={styles.headerText}>
-                            Answer The Question
+                            Answer the Question
```

Comment:
Should the question also be lower case?

Reply:
Fair point but I think we're following the figma (https://www.figma.com/file/PadzU8GqyhFLCjIt7rQkpE/RightOn-Apps?node-id=7557%3A221842&t=d8m7J8se2OL3Onr4-0):
![image](https://user-images.githubusercontent.com/79417944/211907116-21517971-5854-40f7-a1a0-d6e119991af9.png)

So I'd like to just mark this as resolved. I will merge now but let me know if you all see anything else.

Reply:
Figma has lowercase 'the' for the header